src/test/kotlin/nl/info/client/pabc/PabcClientServiceTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.pabc
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
+import io.kotest.matchers.shouldBe
+import io.mockk.checkUnnecessaryStub
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.slot
+import nl.info.client.pabc.model.createApplicationRolesResponse
+import nl.info.client.pabc.model.generated.GetApplicationRolesRequest
+
+class PabcClientServiceTest : BehaviorSpec({
+
+    val pabcClient = mockk<PabcClient>()
+
+    val roles = listOf(
+        "behandelaar",
+        "coordinator",
+        "beheerder",
+        "recordmanager",
+        "raadpleger"
+    )
+
+    val mockResponse = createApplicationRolesResponse()
+
+    beforeEach {
+        checkUnnecessaryStub()
+    }
+
+    Given("invoke client with list of functional roles") {
+        val requestSlot = slot<GetApplicationRolesRequest>()
+        every {
+            pabcClient.getApplicationRolesPerEntityType(capture(requestSlot))
+        } returns mockResponse
+
+        val service = PabcClientService(pabcClient)
```

Comment:
Better to create this at the top of the test class. Because otherwise every new test here will also have to create it and that's not necessary. See how we do this for all our other tests.

Reply:
And as variabele or value always use the name of the class, unless there is good reason not to. So `pabcClientService` in this case. This just makes the code more readable. Also for our AI friends.